extensions/positron-r/src/provider.ts

```diff
@@ -58,6 +58,11 @@ export async function* rRuntimeDiscoverer(): AsyncGenerator<positron.LanguageRun
 		binaries.add(b);
 	}
 
+	const pathBinary = await findRBinaryFromPATH();
+	if (pathBinary) {
+		binaries.add(pathBinary);
+	}
+
```

Comment:
As it stands `findRBinaryFromPATH()` will always run (at least) twice: once here and once inside `findCurrentRBinary()`. I have some ideas for a larger refactoring that I will implement when I deal with #3787. I think it's fine to get this new behaviour for the current milestone and streamline things a bit for the next.